spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -61,13 +70,15 @@
       expect(response).to be_redirect
       expect(GenericWork.find(one.id).subject).to eq ["zzz"]
       expect(GenericWork.find(two.id).subject).to eq ["zzz"]
+      expect(GenericWork.find(three.id).subject).not_to eq ["zzz"]
```

Comment:
Negative assertions are not nearly as specific as positive assertions.  Is there a reason you can't do `expect(GenericWork.find(three.id).subject).to eq ["whavever value it initially had"]`?